packages/opus/opus.0.1.2/opam

```diff
@@ -23,6 +23,7 @@ depexts: [
   ["opus-devel"] {os-distribution = "fedora"}
   ["opus-devel"] {os-family = "suse"}
   ["libopus"] {os-distribution = "nixos"}
```

Comment:
Could you remove the `nixos` line if it's not used instead? (it's definitly not used by opam-depext)

Reply:
I clarify this in https://github.com/ocaml/opam-repository/pull/14477#issuecomment-510849631
You can see how opam detects distribution/os/os-family here: https://github.com/ocaml/opam/blob/c35eb977d19457d3f13b31c1aa60de4fb210865d/src/state/opamSysPoll.ml

What is the `ID` field in `/etc/os-release` in NixOS?